beetsplug/fetchart.py

```diff
@@ -802,6 +813,7 @@ def __init__(self):
         self.cover_names = list(map(util.bytestring_path, cover_names))
         self.cautious = self.config['cautious'].get(bool)
         self.store_source = self.config['store_source'].get(bool)
+        self.high_resolution = self.config['high_resolution'].get(bool)
```

Comment:
This (`self.high_resolution`) currently seems to be unused. We can either delete it or use it in directly in the code that needs the flag by using `plugin.high_resolution` instead of a `config[...].get(...)` call.